package.json

```diff
@@ -15,6 +15,9 @@
       "jahia": "javascript/apps/remoteEntry.js"
     }
   },
+  "resolutions": {
```

Comment:
not required to force resolutions as 1.2.6 qualifies as ^1.2.0 and ^1.2.5.

Clean way to do this is delete the `minimist` entry in `yarn.lock` then run yarn. This should update minimist version to latest.

Reply:
So I should remove these lines right?
https://github.com/Jahia/security-filter-tools/blob/52106d5e4bedd88568edc5ce7969c54096bd1a1d/yarn.lock#L3165